config/src/main/java/com/typesafe/config/impl/ConfigDocumentParser.java

```diff
@@ -410,9 +425,9 @@ private ConfigNodeComplexValue parseObject(boolean hadOpenCurly) {
                     putBack(t);
                     break;
                 } else if (flavor != ConfigSyntax.JSON && isIncludeKeyword(t)) {
-                    objectNodes.add(new ConfigNodeSingleToken(t));
-                    parseInclude(objectNodes);
-
+                    ArrayList<AbstractConfigNode> includeNodes = new ArrayList<AbstractConfigNode>();
+                    includeNodes.add(new ConfigNodeSingleToken(t));
+                    objectNodes.add(parseInclude(includeNodes));
```

Comment:
oh I completely missed that nextTokenIgnoringWhitespace is adding to that array. I get it now